bluemira/codes/_freecadapi.py

```diff
@@ -660,6 +660,9 @@ def discretize(w: Part.Wire, ndiscr: int = 10, dl: float = None):
     # discretization points array
     output = w.discretize(ndiscr)
     output = vector_to_numpy(output)
+
+    if w.isClosed():
```

Comment:
I think this "trick" should be moved into the `discretize` method of BluemiraWire. `_freecadapi` methods should return, if possible, what is expected doing the same operation in FreeCAD.

Reply:
Yeah, that makes sense - will do

Reply:
I note that the same trick is already used in `discretize_by_edges` in `_freecadapi.py`

Reply:
_youthful mistakes_ :) I still support my first comment idea, so I would propose to change also `discretize_by_edges`.  The problem probably resides in the fact that there is no counterpart in `tools.py`.

Reply:
Farmed this off to #1085 for now. Probably needs a broader treatment rather than a piecemeal approach.